src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -183,6 +183,33 @@ def triggerDeploy(jenkinsUrl, jobName, token, params) {
   sh(script: "curl -k $url")
 }
 
+def releaseExists(containerTag, repoName, token){
+    try {
```

Comment:
I don't think you need the try catch as it's only adding an echo line.  It will be clear which build stage has failed within Jenkins and the whole pipeline is in a try catch block so would be caught at that level.

Reply:
We have tested without the try catch and in the scenario where it is not able to connect to github to check if the release exists it returns a 'false' result so then proceeds to create a new release when in fact the release already exists but it just cant connect to github to verify the release exists.
Therefore do we want to keep the try catch?

Reply:
I see, makes sense to keep it then.

Reply:
Left try catch block in as agreed.